README.md

```diff
@@ -0,0 +1,5 @@
+# 
+Create a backend for the RealWorld app.
+- create new project and bootstrap it with Prettier and Eslint
+- make HTTP server on Fastify;
+- testing server by Jest;
```

Comment:
Do we have these two points in this PR? If no, please remove them for now

Reply:
Done